helper/Data.php

```diff
@@ -154,7 +154,7 @@ public function fetchPaymentMethods(Cart $cart, $language)
         $responseData = "";
         try {
             $responseData = $this->adyenCheckoutService->paymentMethods($adyenFields);
-        } catch (AdyenException $e) {
+        } catch (\Exception $e) {
```

Comment:
Should we maybe keep the AdyenException catch block and then add the parent catch block (instead of replacing) so that we can have a different message for the two cases in order to easily identify if the issue is ours or not?

Reply:
yes it is indeed a good idea :) Although in this case since there is no different logic per exception type I choose to handle all of them in the same catch which here only means logging it.